### PR TITLE
Add support for promoting hardfork packages

### PR DESCRIPTION
**Pull request by dkijania**
Added new packages eligible for update. Currently one can control which packages are updated and if package should be renamed (like removing hardfork suffix from Debian name). 

Tested on promoting hardfork packages here:

https://buildkite.com/o-1-labs-2/promote-package/builds/124

Docker testing TBD

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: buildkite/src/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Buildkite CI
```
Comment (SanabriaRusso) [inline buildkite/src/README.md:1]: thank you very much for this. Long due this doc.

File: buildkite/src/README.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      }
+
+```
+
+Above job definition defines Job which is tagged by Fast and Lint tags. In the next step you can define Filter variant value which fetch only Fast and Lint tags:
+
+```
+let tags: Filter -> List Tag.Type = \(filter: Filter) -> 
+  merge {
+    QuickLints = [ Tag.Type.Fast , Tag.Type.Fast ]
```
Comment (SanabriaRusso) [inline buildkite/src/README.md:108]: Repeated values? Should one of these be `Tag.Type.Lint`?

**Review by SanabriaRusso — APPROVED** [review]
super nice!

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Pull request merged**